File: SDKs/github.com/ghes-2.22/1.1.4/python/src/sdk/models/operations/teams_remove_membership_for_user_in_org.py
from dataclasses import dataclass, field



@dataclass
class TeamsRemoveMembershipForUserInOrgPathParams:
    org: str = field(default=None, metadata={'path_param': { 'field_name': 'org', 'style': 'simple', 'explode': False }})
    team_slug: str = field(default=None, metadata={'path_param': { 'field_name': 'team_slug', 'style': 'simple', 'explode': False }})
    username: str = field(default=None, metadata={'path_param': { 'field_name': 'username', 'style': 'simple', 'explode': False }})
    

@dataclass
class TeamsRemoveMembershipForUserInOrgRequest:
    path_params: TeamsRemoveMembershipForUserInOrgPathParams = field(default=None)
    

@dataclass
class TeamsRemoveMembershipForUserInOrgResponse:
    content_type: str = field(default=None)
    status_code: int = field(default=None)
    

File: SDKs/github.com/ghes-2.21/1.1.4/typescript/src/sdk/models/operations/teamsremovemembershipforuserinorg.ts
import { Metadata, SpeakeasyBase } from "../../../internal/utils/utils";


export class TeamsRemoveMembershipForUserInOrgPathParams extends SpeakeasyBase {
  @Metadata({ data: "pathParam, style=simple;explode=false;name=org" })
  org: string;

  @Metadata({ data: "pathParam, style=simple;explode=false;name=team_slug" })
  teamSlug: string;

  @Metadata({ data: "pathParam, style=simple;explode=false;name=username" })
  username: string;
}


export class TeamsRemoveMembershipForUserInOrgRequest extends SpeakeasyBase {
  @Metadata()
  pathParams: TeamsRemoveMembershipForUserInOrgPathParams;
}


export class TeamsRemoveMembershipForUserInOrgResponse extends SpeakeasyBase {
  @Metadata()
  contentType: string;

  @Metadata()
  statusCode: number;
}
